libraries/llvm/forward-declare-c.ll

```diff
@@ -24,3 +24,13 @@ declare %Pos @c_buffer_show_Char(%Pos)
 declare %Pos @c_buffer_show_Double(%Pos)
 declare %Int @c_buffer_index(%Pos, i64)
 declare %Int @c_buffer_character_at(%Pos, i64)
+
```

Comment:
I don't understand why we always put everything in one file here. Doesn't it make more sense to declare them in array.effekt?